exec/java-exec/src/test/java/org/apache/drill/exec/util/StoragePluginTestUtils.java

```diff
@@ -46,13 +43,6 @@
   public static final String ROOT_SCHEMA = "root";
 
   public static final String DFS_TMP_SCHEMA = DFS_PLUGIN_NAME + "." + TMP_SCHEMA;
-  public static final String DFS_DEFAULT_SCHEMA = DFS_PLUGIN_NAME + "." + SchemaFactory.DEFAULT_WS_NAME;
```

Comment:
Would these changes affect the test unit with marked `@ignore`?

Reply:
No, they shouldn't affect any tests. The logic responsible for creating workspaces for tests was updated and moved from `DrillConnectionImpl` to the `JdbcTestBase` class, and these constants aren't required anymore.